src/content/self-hosted/eks.mdx

```diff
@@ -138,22 +124,18 @@ helm repo update
 
 ### Getting your Okteto License
 
-Okteto is free for small teams.  You get all the features of [Okteto](/docs/enterprise/) for up to 3 users with 3 namespaces each.
+Okteto is free for small teams. You get all the features of [Okteto](/docs/self-hosted/) for up to 3 users with 3 namespaces each.
```

Comment:
I think we should send them to https://okteto.com/free-trial instead, but double check with @bigash42